test/e2e/infibeam-uris-test.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+// set the timeout of these tests to 10 seconds
+jasmine.getEnv().defaultTimeoutInterval = 10000;
+
+const PriceFinder = require('../../lib/price-finder');
+const priceFinder = new PriceFinder();
+
+function verifyPrice(price) {
+  expect(price).toBeDefined();
+
+  // we can't guarantee the price, so just make sure it's a number
+  // that's more than -1
+  expect(price).toBeGreaterThan(-1);
+}
+
+function verifyName(actualName, expectedName) {
+  expect(actualName).toEqual(expectedName);
+}
+
+function verifyCategory(actualCategory, expectedCategory) {
+  expect(actualCategory).toEqual(expectedCategory);
+}
+
+describe('price-finder for Infibeam Store URIs', () => {
+  describe('testing [ENTER ITEM NAME HERE] item', () => {
+    // TODO provide URI and contents for test here!
```

Comment:
A minor point, but could you please update the test name (remove [ENTER ITEM NAME HERE]) and remove the TODO comment.